### PR TITLE
Add `dsmrreader.subdomain.conf.sample`

### PR DESCRIPTION
**Pull request by golles**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/reverse-proxy-confs/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description
Subdomain configuration for [DSMR-reader](https://github.com/dsmrreader/dsmr-reader)

## Benefits of this PR and context

This is an addition to your amazing collection of configurations. I wanted to share this one

## How Has This Been Tested?
I've been using this config since Dec 12, 2020.

## Source / References

https://github.com/dsmrreader/dsmr-reader

### CODE REVIEW

**Comment by golles** [comment]
Feel free to decline this PR if you think it has no added value.

File: dsmrreader.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    include /config/nginx/ssl.conf;
+
+    client_max_body_size 0;
+
+    # enable for ldap auth (requires ldap-location.conf in the location block)
+    #include /config/nginx/ldap-server.conf;
+
+    # enable for Authelia (requires authelia-location.conf in the location block)
+    include /config/nginx/authelia-server.conf;
```
Comment (nemchik) [inline dsmrreader.subdomain.conf.sample:19]:
```suggestion
    #include /config/nginx/authelia-server.conf;
```

File: dsmrreader.subdomain.conf.sample [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    location / {
+        # enable the next two lines for http auth
+        #auth_basic "Restricted";
+        #auth_basic_user_file /config/nginx/.htpasswd;
+
+        # enable for ldap auth (requires ldap-server.conf in the server block)
+        #include /config/nginx/ldap-location.conf;
+
+        # enable for Authelia (requires authelia-server.conf in the server block)
+        include /config/nginx/authelia-location.conf;
```
Comment (nemchik) [inline dsmrreader.subdomain.conf.sample:33]:
```suggestion
        #include /config/nginx/authelia-location.conf;
```

**Review by nemchik — CHANGES_REQUESTED** [review]
Auth lines need to be commented by default. Otherwise it looks good to me.

**Comment by golles** [comment]
@nemchik Thank you for the feedback, I missed restoring these lines from how I use them.
I think I screwed up both PR's. I deleted the fork of this repo (`golles closed this by deleting the head repository on Feb 23`), I was able to reopen the PR's by forking it again, however, I don't have the tracking branches any longer.
Usually, there is a commit suggestion next to a code suggestion comment.

I think I have to recreate new PR's for this, I will do this soon

**Comment by golles** [comment]
That was a nice idea, but that didn't work, as there is no place where I can cherry-pick that commit from. I've created #553 te replace this PR, sorry for the trouble

**Pull request closed without merging**